arbeitszeit_flask/company/routes.py

```diff
@@ -273,27 +269,22 @@ class request_cooperation(RequestCooperationView): ...
 @commit_changes
 def my_cooperations(
     list_coordinations: ListCoordinationsOfCompany,
-    list_inbound_coop_requests: ListInboundCoopRequests,
-    list_outbound_coop_requests: ListOutboundCoopRequests,
+    show_company_cooperations: ShowCompanyCooperationsUseCase,
     list_my_cooperating_plans: ListMyCooperatingPlansUseCase,
     presenter: ShowMyCooperationsPresenter,
 ):
     list_coord_response = list_coordinations(
         ListCoordinationsOfCompanyRequest(UUID(current_user.id))
     )
-    list_inbound_coop_requests_response = list_inbound_coop_requests(
-        ListInboundCoopRequestsRequest(UUID(current_user.id))
-    )
-    list_outbound_coop_requests_response = list_outbound_coop_requests(
-        ListOutboundCoopRequestsRequest(UUID(current_user.id))
+    use_case_response = show_company_cooperations.show_company_cooperations(
```

Comment:
Minor. The wording `use_case_response` seems a little generic the other to response are use case responses as well.